includes/definitions/discovery/vrp.yaml

```diff
@@ -30,7 +30,15 @@ modules:
                     num_oid: '.1.3.6.1.4.1.2011.5.25.31.1.1.3.1.5.{{ $index }}'
                     descr: '{{ $entPhysicalName }}'
                     index: '{{ $index }}'
-                    skip_value_lt: -50
+                    skip_values: 
+                        - 
+                            oid: hwEntityOpticalMode
+                            op: '='
+                            value: '1'
+                        - 
+                            oid: hwEntityOpticalTemperature
+                            op: '<'
+                            value: '-50'
```

Comment:
the AC6605 does add 'garbage' values to the table if the sensors are meaningless, so more filtering is necessary. Does not break existing families (as far as Travis, and my production says)